File: CSharpFundamentals/04_ConsoleInputAndOutput/06_QuadraticEquation/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main()
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double d = 0;

      //  if (b % 2 == 0)
      //  {
      //      double k = b / 2;
      //      d = (Math.Pow(k, 2)) - 4 * a * c;
      //
      //      if (d < 0)
      //      {
      //          Console.WriteLine("no real roots");
      //      }
      //      if (d == 0)
      //      {
      //          double x = -k / a;
      //          Console.WriteLine("{0:f2}", x);
      //      }
      //      if (d > 0)
      //      {
      //          double x1 = (-k + Math.Sqrt(d)) / a;
      //          double x2 = (-k - Math.Sqrt(d)) / a;
      //          if (x1 < x2)
      //          {
      //              Console.WriteLine("{0:f2}", x1);
      //              Console.WriteLine("{0:f2}", x2);
      //          }
      //          else
      //          {
      //              Console.WriteLine("{0:f2}", x2);
      //              Console.WriteLine("{0:f2}", x1);
      //          }
      //      }
      //  }

        d = (Math.Pow(b, 2)) - (4 * a * c);

        if (d < 0)
        {
            Console.WriteLine("no real roots");
        }
        if (d == 0)
        {
            double x = -b / (2 * a);
            Console.WriteLine("{0:f2}", x);
        }
        if (d > 0)
        {
            double x1 = (-b + Math.Sqrt(d)) / (2 * a);
            double x2 = (-b - Math.Sqrt(d)) / (2 * a);
            if (x1 < x2)
            {
                Console.WriteLine("{0:f2}", x1);
                Console.WriteLine("{0:f2}", x2);
            }
            else
            {
                Console.WriteLine("{0:f2}", x2);
                Console.WriteLine("{0:f2}", x1);
            }

        }

    }
}

File: JavaScript/JavaScriptFundamentals/05_ConditionalStatements/QuadraticEquation.js
function solve(args) {
    var a = +args[0];
    var b = +args[1];
    var c = +args[2];
    var d = 0;

    d = (Math.pow(b, 2)) - (4 * a * c);

    if (d < 0)
    {
        console.log("no real roots");
    }
    if (d == 0)
    {
        var x = -b / (2 * a);
        console.log('x1=x2=' + x.toFixed(2));
    }
    if (d > 0)
    {
        var x1 = (-b + Math.sqrt(d)) / (2 * a);
        var x2 = (-b - Math.sqrt(d)) / (2 * a);
        if (x1 < x2)
        {
            console.log('x1=' + x1.toFixed(2) + '; x2=' + x2.toFixed(2));
        }
        else
        {
            console.log('x1=' + x2.toFixed(2) + '; x2=' + x1.toFixed(2));

        }

    }
}

solve([1, 1, 1]);
//solve(['11', '4']);
//solve(['17', '17']);